content/components/subhead.mdx

```diff
@@ -6,10 +6,8 @@ railsIds:
 ---
 
 import ComponentLayout from '~/src/layouts/component-layout'
-import {Text} from '@primer/react'
+import {Link, Text} from '@primer/react'
+import {Link as GatsbyLink} from 'gatsby'
 export default ComponentLayout
 
-<Note variant="warning">
-  <Text sx={{display: 'block', fontWeight: 'bold', mb: 2}}>Work in progress</Text>
-  We are currently working on general guidance for this component.
-</Note>
+For subhead component guidelines, see [pagehead](/components/pagehead).
```

Comment:
Should we create an issue to rename Subhead in PVC or Pagehead in PRC? If so, what should we call this thing? cc @langermank again because of her experience with component API consistency work

Reply:
I'd vote to move forward with Pagehead for all..but agreed it should be dealt with